miso-web/src/main/resources/internal/miso.properties

```diff
@@ -59,6 +59,9 @@ miso.notification.interop.enabled:false
 miso.notification.server.host:your.notification.server
 miso.notification.server.port:8787
 
+# A white-space delimited list of run scanner instances to poll
+miso.runscanner.urls:http://localhost:8080/runscanner
```

Comment:
Well, I'd like to fix 2. Run scanner is going to need a bunch of docs, but I want to wait until it's deployed and working and we've tinkered with it.